src/main/java/org/variantsync/diffdetective/variation/diff/DiffType.java

```diff
@@ -134,17 +134,15 @@ public DiffType inverse() {
     /**
      * Parses the diff type from a line taken from a text-based diff.
      * @param line A line in a patch.
-     * @return The type of edit of <code>line</code> or null if its an invalid diff line.
+     * @return The type of edit of <code>line</code>.
      */
     public static DiffType ofDiffLine(String line) {
         if (line.startsWith(ADD.symbol)) {
             return ADD;
         } else if (line.startsWith(REM.symbol)) {
             return REM;
-        } else if (line.startsWith(NON.symbol)) {
-            return NON;
         } else {
-            return null;
+            return NON;
```

Comment:
I do not know the reasoning behind the change, because it was done by you in commit 438360675c2a0aac9e0f1fe67f4a813c7f8bacc1

Reply:
Maybe returning null previously caused null-pointer exceptions somewhere? 🤔

Reply:
I found that this commit is from an old, closed PR #75  and that Benjamin also pointed out that it should be reverted!

Reply:
All of this was already handled in PR #87 but some duplicate commits show up here again, probably because if rebasing the other PRs.

Reply:
Ahh... that is unfortunate. I think I required the commits, because the bug fixes were important for my use case. At least #75 sounds important for me. If the commits are duplicated, merging the branches should resolve this. However, I am not sure that I have merged develop into my branch correctly.

Reply:
Paul and I discussed this problem and we decided to open a new cleaned-up branch, because of the somewhat broken history. The new branch comprises all relevant changes of this pull request, but no undesired ones.